understanding/21/reflow.html

```diff
@@ -39,11 +39,6 @@ <h3>Content exceptions for reflow</h3>
 <p>Complex data tables have a two-dimensional relationship between the headings and data cells. This relationship is essential to convey the content. This Success Criterion therefore does not apply to data tables.</p>
       <p>Interfaces which provide toolbars to edit content need to show both the content and the toolbar in the viewport. Depending on the number of toolbar buttons, the toolbar may need to scroll in the direction of text (e.g., horizontally in a vertically scrolling page). This Success Criterion therefore does not apply to interfaces which provide toolbars.</p>
     	</section>
-       <section>
-      <h3>Browsers in mobile operating systems</h3>
-      <p>The layout method of most common browsers in mobile operating systems does not offer reflow in the same way as on desktop browsers. In these browsers, content can be magnified in different ways, for example, by using a pinch gesture to scale up content, or a double tap on a particular column to make it fill the viewport width. This means that currently, mobile browsers usually exhibit horizontal scrolling regardless of what an author does.</p>
```

Comment:
"browsers in mobile operating systems" perhaps better "browsers *on* mobile operating systems"

Reply:
I have no issue changing that, although it is pre-existing language used in both the current heading and the text.

Reply:
Made the changes. Please check and resolve if it looks good.

Reply:
I don't mind removing the reference to the Dolphin browser to make this more future-proof.

> Mobile user agents _can_ in principle offer reflow when users zoom into content [Dolphin reference deleted]. The lack of magnified reflow support in mobile operating systems can therefore be regarded as a user agent support issue.</p>